appveyor.yml

```diff
@@ -34,7 +40,7 @@ test_script:
   - ruby --version && gem --version
   # We test multiple Windows shells because of prior stdout buffering issues
   # filed against Grunt. https://github.com/joyent/node/issues/3584
-  - ps: "npm test # PowerShell" # Pass comment to PS for easier debugging
+  #- ps: "npm test # PowerShell" # Pass comment to PS for easier debugging
```

Comment:
This breaks the build on AppVeyor for 0.10 at least so I had to disable it. It's like it catches the error code of compass and marks the build as errored.

Example https://ci.appveyor.com/project/gruntjs/grunt-contrib-compass/build/18/job/ubbxjala15p3re52